journal/modernized/32-epistles.adoc

```diff
@@ -1201,6 +1209,7 @@ My sincere love, in the unchangeable truth, is to you all, and my prayers to God
 [.signed-section-signature]
 William Edmundson
 
+[.letter-heading]
 === Concerning Men`'s and Women`'s Meetings.
```

Comment:
You just need to remove the === before the Concerning Men`'s and Women`'s Meetings, because its a letter heading. 👍